src/js/edu-benefits/5495/helpers.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import _ from 'lodash/fp';
+import { transformForSubmit } from '../../common/schemaform/helpers';
+
+export const bankAccountChangeLabels = {
+  startUpdate: 'Start or update direct deposit',
+  stop: 'Stop direct deposit',
+  noChange: 'No change to payment method'
+};
+
+export function transform(formConfig, form) {
+  // All the king's horses and all the king's men
```

Comment:
An excellent point. Cleaned up!

If we do find we need it, this form was duplicated from the 1995, so we'll find this code in there.